org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/utils/client/FHIRToolingClient.java

```diff
@@ -85,9 +85,10 @@
 	public static final String DATE_FORMAT = "yyyy-MM-dd";
 	public static final String hostKey = "http.proxyHost";
 	public static final String portKey = "http.proxyPort";
-  private static final int TIMEOUT_NORMAL = 1;
-  private static final int TIMEOUT_OPERATION = 2;
-  private static final int TIMEOUT_OPERATION_LONG = 3;
+  private static final int TIMEOUT_NORMAL = 15;
+  private static final int TIMEOUT_OPERATION = 30;
+  private static final int TIMEOUT_OPERATION_LONG = 60;
+  private static final int TIMEOUT_OPERATION_EXPAND = 120;
```

Comment:
This is also in a couple places, might be good to put it in an enum or const as well.